README.md

```diff
@@ -21,6 +21,44 @@ Available requests described in [this repository](https://gitlab.exactpro.com/vi
 
 This block describes the configuration for check1.
 
+# Example
```

Comment:
Let's move to the end of README
I think a full example should be placed after all descriptions as a result of readme